README.md

```diff
@@ -57,36 +51,47 @@ Package Name | Minimum Version
 
 Package Name | Minimum Version
 ------------ |:--------------:
-[sphinx](http://www.sphinx-doc.org/) | 1.4
-[inkscape](http://inkscape.org) |
+[`sphinx`](http://www.sphinx-doc.org/) | 1.8
 
-> **Note:** We recommend using [pip](https://pypi.org/project/pip) to manage Python modules.
-
-Python Module |
----------- |
-sphinx_rtd_theme |
-recommonmark |
+Python Module | Required version
+------------- |:---------------:
+`sphinx_rtd_theme` | 0.4.3
+`recommonmark` | 0.5.0
+`docutils` | 0.12
 
 
 ### Build process ###
 
-Gaffer also depends on a number of 3rd-party libraries and python modules, many of which are not entirely straightforward to build. We therefore recommend using the latest pre-built dependencies from the [Gaffer dependencies project](https://github.com/GafferHQ/dependencies/releases). These are used in our automated test builds and so are guaranteed to be up-to-date with Gaffer's requirements.
+```bash
+git clone https://github.com/GafferHQ/gaffer.git
+cd gaffer
+```
+
+Gaffer depends on a number of 3rd-party libraries. We recommend using the latest pre-built dependencies from the [Gaffer dependencies project](https://github.com/GafferHQ/dependencies/releases). These are used in our automated test builds and so are guaranteed to be up-to-date with Gaffer's requirements.
 
-Once you've downloaded the dependencies, unpack them and move/rename them to the directory in which you want to make your Gaffer build. We'll refer to this location as `<BUILD_DIR>` in the instructions below. Before continuing, make sure the dependencies are unpacked as `<BUILD_DIR>/include`, `<BUILD_DIR>/lib`, etc.
+The dependencies distribution forms the basis of a Gaffer build and the root of the expanded archive becomes the `BUILD_DIRECTORY` we pass to `scons`.
 
-Next, clone the Gaffer source and `cd` into its directory:
+The easiest way to get them is with the `config/installDependencies.sh` script included in the Gaffer source tree. This downloads the correct version for the version of Gaffer you are building and unpacks them to a directory of your choice.
+
+In this example we're going to build gaffer to a `gaffer-build` directory next to our checkout.
```

Comment:
next to our checkout.
↓
in parallel with our source directory.

Reply:
Im not sure about `parallel` as it perhaps infers some kind of continuity of directory structure, which there isn't really. I think as it stands is a fairly common way to phrase it in dev world.

Reply:
Using "checkout" as a noun made me do a double-take, but if you insist...